demo/demo.md

```diff
@@ -140,6 +140,90 @@ Note: Setting the `value` to `undefined` will also reset the element.
+
+#### Date Example
+
+<div class="exampleWrapper exampleWrapper--flex">
+  <auro-input id="demoMaxDate" type="month-day-year" max="03/25/2023" setCustomValidityRangeOverflow="The selected date is past the defined maximum date.">
+    <span slot="label">Choose a date</span>
+  </auro-input>
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+  ```html
+  <auro-input id="demoMaxDate" type="month-day-year" max="03/25/2023" setCustomValidityRangeOverflow="The selected date is past the defined maximum date.">
+    <span slot="label">Choose a date</span>
+  </auro-input>
+  ```
+</auro-accordion>
+
+#### Numeric Example
+
+<div class="exampleWrapper exampleWrapper--flex">
+  <auro-input id="demoMaxNumber" type="numeric" max="10" setCustomValidityRangeOverflow="The selected value is above the defined maximum.">
+    <span slot="label">Choose a number</span>
+  </auro-input>
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+  ```html
+  <auro-input id="demoMaxNumber" type="numeric" max="10" setCustomValidityRangeOverflow="The selected value is above the defined maximum.">
+    <span slot="label">Choose a number</span>
+  </auro-input>
+  ```
+</auro-accordion>
+
+### min
+
+Use the `min` attribute to define a minimum value used during validation. The attribute will only apply when `auro-input` also has a `type` attribute for `numeric` or any date format.
+
+The `min` attribute should be used in combination with the `setCustomValidityRangeUnderrflow` attribute to define help text used when the `min` attribute validation fails.
+
+#### Date Example
+
+<div class="exampleWrapper exampleWrapper--flex">
+  <auro-input id="demoMinDate" type="month-day-year" min="03/25/2023" setCustomValidityRangeUnderflow="The selected date is before the defined minimum date.">
+    <span slot="label">Choose a date</span>
+  </auro-input>
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+  ```html
+  <auro-input id="demoMinDate" type="month-day-year" min="03/25/2023" setCustomValidityRangeUnderflow="The selected date is before the defined minimum date.">
+    <span slot="label">Choose a date</span>
+  </auro-input>
+  ```
+</auro-accordion>
+
+#### Numeric Example
+
+<div class="exampleWrapper exampleWrapper--flex">
+  <auro-input id="demoMinNumber" type="numeric" min="10" setCustomValidityRangeUnderflow="The selected value is below the defined minimum.">
+    <span slot="label">Choose a number</span>
+  </auro-input>
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+  ```html
+  <auro-input id="demoMinNumber" type="numeric" min="10" setCustomValidityRangeUnderflow="The selected value is below the defined minimum.">
+    <span slot="label">Choose a number</span>
+  </auro-input>
+  ```
+</auro-accordion>
+
 ### Pattern
```

Comment:
I don't think the `setCustomValidity` example works as intended. The only way to trigger the error message is by clicking in and out of the input without typing anything, which is misleading due to the error message. The validity works as intended and is not included in this PR, but I wanted to bring that to your attention.